includes/wizards/class-reader-revenue-wizard.php

```diff
@@ -508,16 +550,41 @@ public function api_check_salesforce_token() {
 	 * @return Array
 	 */
 	protected function fetch_all_data() {
+		$platform                 = get_option( NEWSPACK_READER_REVENUE_PLATFORM );
```

Comment:
Is there any chance this option could be undefined at this point, and would that cause any issues with no default value? It looks like this endpoint would basically return the default `$args`, right?

Reply:
Since this function is only used by the Reader Revenue Wizard, I think the worst case scenario would be some unusual results in the later screens of the wizard, but definitely worth fixing. Done in https://github.com/Automattic/newspack-plugin/pull/726/commits/6d9aaafc91100ca9f384650e953b2f931d5cf122